ansible/configs/ocp-clientvm-rhel8/files/cloud_providers/ec2_cloud_template.j2

```diff
@@ -0,0 +1,356 @@
+#jinja2: lstrip_blocks: True
```

Comment:
for something as simple as clientvm, do we really need this template file ?
Can we use the default template instead ?

Reply:
I agree. Change pushed.